servers/rendering/renderer_scene_cull.cpp

```diff
@@ -2591,7 +2617,23 @@ void RendererSceneCull::render_camera(const Ref<RenderSceneBuffers> &p_render_bu
 						camera->znear,
 						camera->zfar,
 						camera->vaspect);
+			} break;
+			case Camera::OBLIQUE: {
+				Quaternion oblique_plane = get_camera_oblique_quaternion(p_camera);
```

Comment:
If this is a plane, not a quaternion, then the function should be named `get_camera_oblique_plane` and the return type should be `Plane`, not `Quaternion`.

If Godot's Plane type is somehow not suitable for this case, then it should use Vector4 rather than Quaternion.

Reply:
The drawback to using a plane is that must be converted to quaternion before it can be used in the step `p_oblique_plane * (2.0F / p_oblique_plane.dot(q))` because the Plane type does not support mathematical operations.

Quaternion was used because at the time (almost 3 years ago) that this camera was originally implemented the Vector4 type did not exist.